packages/client/test/test-utils/fake/FakeChain.ts

```diff
@@ -12,9 +14,56 @@ export interface StreamRegistryItem {
     permissions: Multimap<EthereumAddress | PublicPermissionTarget, StreamPermission>
 }
 
+export interface Events {
+    streamAddedToStorageNode: (payload: Omit<StorageNodeAssignmentEvent, 'blockNumber'>) => void
```

Comment:
Better to just emit the whole thing but always use `blockNumber: 0`?
```suggestion
    streamAddedToStorageNode: (payload: StorageNodeAssignmentEvent) => void
```

Reply:
Or maybe a separate interface for the payload as this is not related to the `StreamrClient` event?